stl/inc/numeric

```diff
@@ -625,6 +625,18 @@ _NODISCARD constexpr common_type_t<_Mt, _Nt> gcd(const _Mt _Mx, const _Nt _Nx) n
     using _Common          = common_type_t<_Mt, _Nt>;
     using _Common_unsigned = make_unsigned_t<_Common>;
 
+    if constexpr (is_signed_v<_Common>) {
+#ifndef _DEBUG
+        if (_STD _Is_constant_evaluated())
+#endif // ^^^ !defined(_DEBUG) ^^
+        {
+            if (_Mx == _STD _Min_limit<_Common>() || _Nx == _STD _Min_limit<_Common>()) {

```

Comment:
Suggestion: We could extract `_Min_limit<_Common>()` into a `constexpr` variable.